examples/js/loaders/sea3d/physics/SEA3DAmmoLoader.js

```diff
@@ -491,7 +491,7 @@ THREE.SEA3D.Domain.prototype.applyTransform = function ( matrix ) {
 		// ignore rotation scale
 
 		mtx.scale( vec.set( 1 / obj3d.scale.x, 1 / obj3d.scale.y, 1 / obj3d.scale.z ) );
-		obj3d.rotation.setFromRotationMatrix( mtx );
+		obj3d.quaternion.setFromRotationMatrix( mtx );
```

Comment:
/ping @sunag Perhaps this block of code can be implemented with this, instead:

```javascript
obj3d.applyMatrix( matrix );
```

Reply:
Hi @WestLangley 
I do not know now but I broke my head a little because quaternion is affected in decomposition matrix if scale is not  (1,1,1)
https://github.com/WestLangley/three.js/blob/e865ad729f60fe919373a14073a9486c90a92d9c/examples/js/loaders/sea3d/physics/SEA3DAmmoLoader.js#L493

Reply:
in AS3(Flash) version for example the decompose matrix scale not influence the quaternion.